girder/cumulus/server/models/volume.py

```diff
@@ -109,6 +111,16 @@ def append_to_log(self, user, id, record):
         self.update({'_id': ObjectId(id)}, {'$push': {'log': record}})
         send_log_notification('volume', volume, record)
 
+    def update_volume(self, user, volume):
```

Comment:
fixed this by amending last commit, server/volume.py uses send_status_notification for detach_complete